screens/Home/Home.js

```diff
@@ -1,56 +1,81 @@
-import React from 'react';
+import React, { useState } from 'react';
 import gql from 'graphql-tag';
 import { useQuery } from '@apollo/react-hooks';
-import { StyleSheet, View, Text, Image, TouchableOpacity } from 'react-native';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  ScrollView,
+  RefreshControl,
+} from 'react-native';
 import { removeToken } from '../../AuthManager';
+import AudioPlayer from '../../elements/AudioPlayer/AudioPlayer.js';
+
+import Logo from '../../assets/Logo.svg';
+import styles from '../../styles/homeStyle';
+import universalStyle from '../../styles/styles';
+
 
 const GET_HOME_DATA = gql`
-  query LatestPosts {
-    latestPosts {
-      id
-    }
+  query HomeData {
     userInfo {
-      email
+      id
       username
     }
+    latestPosts {
+      id
+      audioUrl
+      author {
+        id
+        username
+      }
+    }
   }
 `;
 
 export function Home() {
-  const { loading, error, data } = useQuery(GET_HOME_DATA);
+  const { loading, error, data, refetch } = useQuery(GET_HOME_DATA);
+
+  const [refreshing, setRefreshing] = useState(false);
+
+  const _onRefresh = () => {
```

Comment:
```suggestion
  const onRefresh = () => {
```

I metodi prefixed con `_` sono una convenzione in JavaScript per denotare metodi privati: https://javascript.info/private-protected-properties-methods

In questo caso non serve comunque